docs/glossary.md

```diff
@@ -0,0 +1,202 @@
+---
+id: glossary
+title: Cumulus Glossary
+hide_title: true
+---
+
+# AWS Glossary
+
+For terms/items from Amazon/AWS not mentioned in this glossary, please refer to the [AWS Glossary](https://docs.aws.amazon.com/general/latest/gr/glos-chap.html).
+
+# Cumulus Glossary of Terms
+
+### API Gateway
+
+  A Cumulus component that provides an API that provides accessors to Cumulus functionality.
+
+  This component is published as a [NPM package](https://www.npmjs.com/package/@cumulus/apiAPI).   API documentation can be viewed [here](https://nasa.github.io/cumulus-api/).
```

Comment:
This link is incorrect. Should be https://www.npmjs.com/package/@cumulus/api

Reply:
03e1647c